config/locales/translations/zendesk_apps_support.yml

```diff
@@ -254,6 +254,10 @@ parts:
       key: "txt.apps.admin.error.app_build.translation.missing_required_key"
       title: "App builder job: required key missing from translation file"
       value: "Missing required key from %{file}: %{missing_key}"
+  - translation:
+      key: "txt.apps.admin.error.app_build.translation.missing_required_key_on_leaf"
+      title: "App builder job: required key missing from translation file"
+      value: "Missing required key %{missing_key} on leaf %{leaf} from %{file}"
```

Comment:
@hiroiguchi please add info/screenshots or translations will come back funny for sure ;-)

Reply:
guys it is better to create separate PR for strings only so they can be merged quicker than the code (since translation take time) - best practice only. Thanks!

Reply:
@hiroiguchi I added a screenshot. would that be enough?